docs/getting_started/install_ez_platform.md

```diff
@@ -76,13 +76,13 @@ Log in to your service portal on [support.ibexa.co](https://support.ibexa.co), g
     composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
     ```
 
-    To store your credentials per project, add the credentials to the global `COMPOSER_AUTH` variable:
+    To store your credentials per project, add the credentials to the `COMPOSER_AUTH` variable:
 
     ``` bash
     export COMPOSER_AUTH='{"http-basic":{"updates.ibexa.co": {"username": "<your-key>", "password": "<your-password>"}}}'
     ```
 
-    Then, add this global variable to `auth.json`.
+    Then, add this variable to `auth.json` (for example, by running `echo COMPOSER_AUTH > auth.json`).
```

Comment:
That's not the point of the issue :confused: 

This file `auth.json` has to be created **after** the call to `composer create-project`. Otherwise it complains that the directory is not empty.